node/blockchain/tests/test_models/test_signed_change_request/test_node_declaration.py

```diff
@@ -24,18 +26,61 @@ def test_create_from_node_declaration_signed_change_request_message(
     )
 
 
-@pytest.mark.skip('Not implemented yet')
-def test_serialize_and_deserialize_node_declaration():
-    # TODO(dmu) HIGH: Implement similar to `.test_genesis.test_serialize_and_deserialize_genesis_type`
-    #                 https://thenewboston.atlassian.net/browse/BC-169
-    raise NotImplementedError
+def test_serialize_and_deserialize_node_declaration(
+    node_declaration_signed_change_request_message, regular_node_key_pair
+):
+    signed_change_request = SignedChangeRequest.create_from_signed_change_request_message(
+        message=node_declaration_signed_change_request_message,
+        signing_key=regular_node_key_pair.private,
+    )
+    assert isinstance(signed_change_request, NodeDeclarationSignedChangeRequest)
+    serialized = signed_change_request.json()
+    deserialized = SignedChangeRequest.parse_raw(serialized)
+    assert isinstance(deserialized, SignedChangeRequest)
```

Comment:
Why not?

`assert isinstance(deserialized, NodeDeclarationSignedChangeRequest)`

Reply:
I fixed that.